### PR TITLE
migration 19 fix

### LINKED ISSUE #2809: migration 19 throws error when runed from yarn hot
when starting uwazi, files collection on the database is created and after that migration 19 tries to rename uploads to files, at this point files already exists and throws an error.

### PR DESCRIPTION
**Pull request by daneryl**
fixes #2809 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/api/migrations/migrations/19-rename-uploads-to-files/specs/19-rename-uploads-to-files.spec.js [hunk 1]
```diff
@@ -41,4 +41,9 @@ describe('migration rename-uploads-to-files', () => {
     await testingDB.mongodb.collection('uploads').drop();
     await migration.up(testingDB.mongodb);
   });
+
+  it('should not fail when uploads collection does not exist', async () => {
```
Comment (RafaPolit) [inline app/api/migrations/migrations/19-rename-uploads-to-files/specs/19-rename-uploads-to-files.spec.js]: This description does not match the test.

File: app/api/migrations/migrations/19-rename-uploads-to-files/index.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ export default {
   async up(db) {
     process.stdout.write(`${this.name}...\r\n`);
 
+    try {
+      await db.collection('files').drop();
```
Comment (RafaPolit) [inline app/api/migrations/migrations/19-rename-uploads-to-files/index.js]: This is not idempotent, which has been an 'unspoken' rule of migrations.  Not sure if it merits here to go the extra effort to ensure that a second run will not forever delete the already-migrated files?

Comment (daneryl) [reply app/api/migrations/migrations/19-rename-uploads-to-files/index.js]: @RafaPolit changed it so it only runs the migration when has not ran before, this assumes that if uploads collection does not exist then the migration has already ran.

**Comment by RafaPolit** [comment]
LGTM

**Pull request merged**